zowe/imperative
# Handle new APIML unique cookie identifier

**What It Does**

- resolves #979
- added a new `add ImperativeExpect.toMatchRegExp` method
- prevented multiple `logout` operations from failing due to not having a token available 
- allowed `autoStore` for dynamic APIML token types
- ~allowed `logout` operation to remove token type and/or token value when either is not specified in the config file~
  - this cannot be done because it conflicts with the ability to logout a token different than the one stored in the vault
- added utility method: `getProfileNameFromPath` to help with nested token authentication
- added support for multiple token authentication processes in a single `config secure` command
  - added an entry to the `console.log` to inform which profile credentials are being prompted for
- prevented `auto-init` from performing two `login` operations on a single command

**How to Test**


I recommend to combine this PR with the CLI counterpart:
  - https://github.com/zowe/zowe-cli/pull/1734

Please see the above PR for testing instructions.

**Review Checklist**
I certify that I have:


**Additional Comments**

Opened issues as a result of this PR:
 - https://github.com/zowe/imperative/issues/995
